kong/runloop/handler.lua

```diff
@@ -78,7 +78,6 @@ local HOST_PORTS = {}
 
```

Comment:
This file is usually a source of many headaches when merging OSS changes into EE, so if you are doing refactor work it is recommended to open a PR against EE as well.

Reply:
Yes, if this PR is merged, I will cherry-pick it into EE.